doc/reference/cmdline.rst

```diff
@@ -52,14 +86,59 @@ Running the server
 
     disables demo data loading for modules installed
     comma-separated, use ``all`` for all modules.
+    Requires :option:`-d` and :option:`-i`.
+
+.. option:: --pidfile=<pidfile>
+
+    path to a file where the server pid will be stored
+
+.. option:: --stop-after-init
+
+    stops the server after its initialization.
+
+.. option:: --osv-memory-count-limit <memory_limit>
+
+   Force a limit on the maximum number of records kept in
+   the virtual osv_memory tables. By default there is no limit.
```

Comment:
Transients have not been virtual tables in a while. Not to mention both this and the next option are conditional on the autovacuum's run, which is daily by default, so the implication that transient records are kept 1h by default is... misleading.

Reply:
description for this and --transient-age-limit  is taken from config.py
```
group.add_option("--osv-memory-count-limit", dest="osv_memory_count_limit", my_default=False,
                         help="Force a limit on the maximum number of records kept in the virtual "
                              "osv_memory tables. By default there is no limit.",
                         type="int")
group.add_option("--transient-age-limit", dest="transient_age_limit", my_default=1.0,
                         help="Time limit (decimal value in hours) records created with a "
                              "TransientModel (mostly wizard) are kept in the database. Default to 1 hour.",
                         type="float")
```
Please advice how to change description, I will change it in both places.

Reply:
I don't have a good docstring at the moment. Removed the mention of these options until we fix the config properly.